# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/shadow/ObjectGroup.h
# Commit: 11fc8f557a2a
# Full Hash: 11fc8f557a2a7cf3c89e7c1396316aabd9725778
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 9bfa6023a553 -r 11fc8f557a2a js/public/shadow/ObjectGroup.h
--- a/js/public/shadow/ObjectGroup.h	Sat Mar 06 01:05:20 2021 +0000
+++ b/js/public/shadow/ObjectGroup.h	Sat Mar 06 01:05:21 2021 +0000
@@ -23,7 +23,6 @@
 struct ObjectGroup {
   int* unused;
   JSObject* proto;
-  JS::Realm* realm;
 };
 
 }  // namespace shadow